src/function-builder.ts

```diff
@@ -284,6 +290,29 @@ export function runWith(runtimeOptions: RuntimeOptions): FunctionBuilder {
   }
 }
 
+function _onCall<T = any, Return = any | Promise<any>>(
```

Comment:
Aside, I'm somewhat disgruntled that we still have `_` prefixes in our codebase. It was the style when programming with Closure, but is not part of the style guide anymore. I'm not sure wether it's worse to propagate the old style or have divergent styles.